README.md

```diff
@@ -1,4 +1,99 @@
 Sailjail permissions
 ====================
-
 Permissions for [Sailjail](https://github.com/sailfishos/sailjail/).
+These are [firejail](https://firejail.wordpress.com/) profile files with
+Sailjail specific metadata format.
+
+Permissions
+-----------
+Permissions that applications may use:
+
+- Accounts
+  - Using accounts, including editing them. Syncing accounts.
+- Ambience
+  - Set and edit ambiences.
+- AppLaunch
+  - Launching and stopping systemd services. This is usually needed for
+    background tasks.
+- ApplicationInstallation
+  - Installing and uninstalling applications.
+- Audio
+  - Playing audio, changing audio configuration and showing audio controls on
+    lockscreen.
+- Bluetooth
+  - Connecting to and using Bluetooth and NFC hardware.
+- Calendar
+  - Display and editing of calendar events.
+- Camera
+  - Access to camera hardware to take photos or video.
+- Contacts
+  - Display and editing of contacts data. Access to contact cards.
+- Documents
+  - Access to Documents directory.
+- Downloads
+  - Access to Downloads directory.
+- Email
+  - Reading and sending emails. Access to email attachments.
+- Internet
+  - Using data connection and connecting to internet.
+- Location
+  - Use GPS and low accuracy positioning.
+- MediaIndexing
+  - Access to Tracker to list files on device. If you have access to data
+    directory, you may want to use also this.
+- Messages
+  - Access to message data and to send SMS messages.
+- Mozilla
+  - If you use Gecko based WebView you need this.
+- Music
+  - Access to Music directory, playlists and coverart cache.
+- Phone
+  - Make Phone calls, either directly or through system voice call UI.
+- Pictures
+  - Access to Pictures directory and thumbnails.
+- Sharing
+  - Use of sharing pages in the application to share data via Bluetooth, email
+    or other accounts.
+- SyncFW
+  - Access to synchronization framework.
+- Videos
+  - Access to Videos directory and thumbnails.
+
+Internal permissions that applications generally should not use directly:
```

Comment:
Also, some of these are mere dummies / placeholders that are not used even internally, e.g. FingerprintSensor.

Should we identify them separately? Or perhaps remove altogether.

Reply:
Doesn't harm much but I do agree that some are more like placeholders.